api-tests/core/admin/ee/review-workflows.test.api.js

```diff
@@ -492,40 +501,26 @@ describeOnCondition(edition === 'EE')('Review workflows', () => {
 
   describe('Deleting a stage when content already exists', () => {
     test('When content exists in a review stage and this stage is deleted, the content should be moved to the nearest available stage', async () => {
-      // Get the default workflow stages
-      const res = await requests.admin.get(`/admin/review-workflows/workflows/1/stages`);
-      const defaultStages = res.body.data;
-
-      const productsBefore = await findAll(productUID);
-      const entriesMovedToEnd = productsBefore.results
-        .filter((entry) => entry.id % 2 === 0)
-        .map((entry) => entry.id);
-
-      await mapAsync(entriesMovedToEnd, async (entityId) =>
-        requests.admin.put(
-          `/admin/content-manager/collection-types/${productUID}/${entityId}/stage`,
-          {
-            body: {
-              data: { id: defaultStages.slice(-1)[0].id },
-            },
-          }
-        )
+      const products = await findAll(productUID);
+
+      // Move the first half of the entries to the last stage,
+      // and the second half to the first stage
```

Comment:
```suggestion
      // Move half of the entries to the last stage,
      // and the other half to the first stage
```

I think it's actually moving every other one rather than the first and last half. wdyt?

Reply:
you are right! :)